go/common/ledger/ledger.go

```diff
@@ -6,9 +6,10 @@ import (
 )
 
 const (
-	// PathPurpose is set to 44 to indicate use of the BIP-0044 specification.
-	PathPurpose uint32 = 44
-
+	// PathPurposeEntity is set to 44 to indicate use of the BIP-0044 specification.
+	PathPurposeEntity uint32 = 44
+	// PathPurposeConsensus is set to 43 to be used with ledger Validator app
+	PathPurposeConsensus uint32 = 43
 	// ListingPathCoinType is set to 474, the index registered to Oasis in the SLIP-0044 registry.
```

Comment:
Then I would add the consensus subpurpose like I mentioned in https://github.com/oasisprotocol/oasis-core/pull/3058#discussion_r448258724:

```go
	// PathSubPurposeConsensus is set to 0 to indicate it is to be used for
	// consensus-related things.
	PathSubPurposeConsensus uint32 = 0
```

Reply:
Added.